src/pattern-library/components/patterns/layout/OverlayPage.js

```diff
@@ -0,0 +1,82 @@
+import { useState } from 'preact/hooks';
+
+import { Button, Card, CardContent, Spinner } from '../../../../next';
+import { Overlay } from '../../../../next';
+import Library from '../../Library';
+import Next from '../../LibraryNext';
+
+export default function OverlayPage() {
+  const [defaultOpen, setDefaultOpen] = useState(false);
+  const [darkOpen, setDarkOpen] = useState(false);
+  const [lightOpen, setLightOpen] = useState(false);
+
+  function toggleDefaultOverlay() {
```

Comment:
I think it is more conventional for us to use arrow functions `const toggleDefaultOverlay = ...` rather than named functions for helpers inside components.